src/components/Thought.tsx

```diff
@@ -536,6 +536,7 @@ const ThoughtContainer = ({
     'child-divider': isDivider(thought!.value ?? ''),
     'cursor-parent': isCursorParent,
     'cursor-grandparent': isCursorGrandparent,
+    [`context-${hashContext(pathToContext(path), rank)}`]: true,
```

Comment:
Yes indeed. We can implement searching ediatble node by text content and then find the associated bullet node. However we won't be able to differentiate between two nodes with same text content.

Reply:
That will work for normal view 99% of the time. For context view we will have to have a different approach since the same thought can appear on the screen more than once. 

I’m flexible here, just wanted you to give it some thought and see if there was an easy way to improve the testing interface.